### PR TITLE
use uuid instead of crypto to generate ids

### PR DESCRIPTION
**Pull request by volodymyrZotov**
Resolves: #73 

*Description*:
Webpack 5 stopped automatically polyfilling Node.js ([src](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#automatic-nodejs-polyfills-removed)). Because we use the node.js crypto library in one area of the item builder, users trying to integrate the SDK with the webpack bundler will see an error.

*Solution*:
Use `uuid` library instead of `crypto` module to generate valid 1password ids.

### CODE REVIEW

**Review by jillianwilson — APPROVED** [review]

**Comment by edif2008** [comment]
From the looks of it we generate IDs only for sections, right? If so, I've got two follow-up questions:
- Why don't we also generate field IDs? If Connect is able to generate IDs for fields, doesn't it also generate IDs for sections?
- If we do need to generate section IDs with the SDK, then the format is not the right one. it should be `Section-<uuid>`. Here's an example:
  ```json
  "sections": [
    {
      "id": "Section_tco6nsqycj6jcbyx63h5isxcny",
      "label": "test-section"
    }
  ],
  ```

File: package.json [hunk 1]
```diff
@@ -42,13 +42,13 @@
...
-        "typescript": "^4.0.3",
-        "slugify": "^1.6.5"
+        "typescript": "^4.0.3"
     },
     "dependencies": {
         "axios": "^0.21.3",
         "debug": "^4.3.1",
         "lodash.clonedeep": "^4.5.0",
-        "slugify": "^1.6.5"
+        "slugify": "^1.6.5",
```
Comment (edif2008) [inline package.json:51]: Why is `slugify` moved from `devDependencies` to `dependencies`? I don't see any direct use of it in the code.

**Review by edif2008 — APPROVED** [review]
This PR is looking good. We've also improved section ID generation, which is a nice touch.
Left one tiny comment that shouldn't block this PR from merging.

File: package.json (hunk 1 above)
Comment (edif2008) [reply package.json:51]: I've noticed just now that it's a duplicate, but the question still remains: Why is it in `dependencies` and not in `devDependencies`?

Comment (volodymyrZotov) [reply package.json:51]: It's automatically changed by `npm` when I installed `uuid` package.

I also do not see `slugify` usage anywhere in the code. 
I think it was added to generate 1Password ids, but after some time we stoped using it and just forgot to remove it.
I'm going to delete it from dependencies at all. Nice catch!

Comment (volodymyrZotov) [reply package.json:51]: It was automatically managed by `npm` when I installed `uuid` package.
There is used within `builders.ts`.
As we use it directly in out code and it won't work without it we need to keep the package in `dependencies` section. Everything that's inside `devDependencies` won't be included in production build.

Comment (edif2008) [reply package.json:51]: Awesome. Thanks for the clarification.

**Pull request merged**